service/policy/db/migrations/20241125220354_keys_table.sql

```diff
@@ -0,0 +1,233 @@
+-- +goose Up
```

Comment:
Could you please add a markdown file `service/policy/db/migrations/20241125220354_keys_table.md` that links to the ADR so we have that historical link between the migration and the relational change?